test/spec-validation.js

```diff
@@ -0,0 +1,40 @@
+const testlab = require('@loopback/testlab');
+const createApp = require('../dist/application').createApp;
+const expect = testlab.expect;
+const validateApiSpec = testlab.validateApiSpec;
+
+describe('Binding', () => {
+  let app;
+  let apiSpec;
+  before(initApp);
+
+  describe('validate Api Spec', () => {
+    const defaultPreamble = {
+      swagger: '2.0',
+      basePath: '/',
+      info: {
+        title: 'LoopBack Application',
+        version: '1.0.0',
+      }
+    };
+
+    it('app.getApiSpec is function', () => {
```

Comment:
This is testing that the LoopBack framework is honoring its API contract. IMO, such test is redundant here - surely the framework has already tested that it provides API as documented.

Please remove.